generated_api_shadow/contrib/envoy/extensions/network/socket_interface/vcl/v3alpha/BUILD

```diff
@@ -0,0 +1,9 @@
+load("@envoy_api//bazel:api_build_system.bzl", "api_proto_package")
```

Comment:
These files should go away now. generated_api_shadow has been removed.

Reply:
Removed